topiary-config/languages_nix.ncl

```diff
@@ -10,6 +10,11 @@
       grammar.source.path = "@css@",
     },
 
+    openscad = {
+      extensions = ["scad"],
+      grammar.source.path = "@openscad@",
+    },
+
```

Comment:
This won't work without the respective update to `default.nix`, where the `openscad` variable should be defined:

https://github.com/mkatychev/topiary/blob/0ce5fa024dd07563fb78f08954c1afbcad0bc925/default.nix#L125-L138

However, this would also depend on your OpenSCAD grammar being packaged in `nixpkgs`, which I presume it isn't.

As such, for now, this ought to be commented out, ideally with a respective comment in `default.nix` (i.e., exactly the same as with `ocamllex`).

```suggestion
    # openscad = {
    #   extensions = ["scad"],
    #   grammar.source.path = "@openscad@",
    # },

```

Reply:
would adding a `flake.nix` to the `tree-sitter-openscad` be of use or does this pipeline have a hard dependency on `nixpkgs`?

Reply:
Yes, I believe this will also work. As you point out, it's how Nickel's grammar is imported by Nix, which also isn't in `nixpkgs` :+1: